src/DataProvider/QueryAwarePartialPaginatorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\DataProvider;
```

Comment:
This file must be moved.

```suggestion
namespace ApiPlatform\Core\Bridge\Doctrine\Orm;
```

Reply:
@teohhanhui Updated